index.html

```diff
@@ -212,72 +224,76 @@
         .sizeByClinicalAttribute("Phos", "Result")
         .orderAllTooltipTables(["Source","Location"])
         .enableTrackTooltips(true)
-        .enableZoom(enableAdvanced) //zoom functionality avaiable only in the advance view
-        .advancedView(enableAdvanced)
-        .enableTrimmedTimeline(true)
-        .enableVerticalLine(false)
-        .enableHelperLines(true)
+        .plugins([
+          {
+            obj: new trimClinicalTimeline("Trim Timeline"), 
```

Comment:
Yes, but I chose to keep it in the constructor because this is something a user could like change according to their preference and hence it should be on the index.html and not hidden deep inside.
Moreover I chose constructor to pass it as the property `name` this is something like the native property of the plugin and not a specification.